### PR TITLE
Update api documentation of function

### PR DESCRIPTION
**Pull request by mfeurer**
This PR just adds the missing functions to the API docs and fixes a few rendering bugs.

### CODE REVIEW

**Comment by amueller** [comment]
wait did we both send the same PR?

File: doc/api.rst [hunk 1]
```diff
@@ -13,11 +13,11 @@ Top-level Classes
...
 
-   OpenMLDataset
-   OpenMLRun
-   OpenMLTask
-   OpenMLSplit
-   OpenMLFlow
+    OpenMLDataset
+    OpenMLRun
+    OpenMLTask
+    OpenMLSplit
```
Comment (amueller) [inline doc/api.rst]: you're missing Evaluation ;)

**Comment by mfeurer** [comment]
Apparently we did ;) Let's just merge both, they're dissimilar enough. And I didn't have evaluations in there one purpose because I wanted to get the functions first.

**Comment by amueller** [comment]
you need to resolve the merge...

**Comment by mfeurer** [comment]
Done ;)

File: doc/api.rst [hunk 2]
```diff
@@ -34,6 +34,30 @@ Top-level Classes
     get_datasets
     list_datasets
 
+:mod:`openml.evaluations`: Evaluation Functions
```
Comment (amueller) [inline doc/api.rst]: I think this section is now duplicate?

**Pull request merged**